man/fifelse.Rd

```diff
@@ -29,12 +29,12 @@ fifelse(dates == "2011-01-01", dates - 1, dates)
 # [1] "2010-12-31" "2011-01-02" "2011-01-03" "2011-01-04" "2011-01-05"
 
 ### Example 2 - Factor class preserved
-v = factor(letters[1:5])
-ifelse(v \%in\% c("a","b","c"), v, factor(NA))
-# [1]  1  2  3 NA NA
-fifelse(v \%in\% c("a","b","c"), v, factor(NA))
-# [1] a    b    c    <NA> <NA>
-# Levels: a b c d e
+v = factor(letters[1:3])
+base::ifelse(c(TRUE,FALSE,TRUE), v, factor("a",levels=levels(v)))
+# [1]  1  1  3
+     fifelse(c(TRUE,FALSE,TRUE), v, factor("a",levels=levels(v)))
+# [1] a    a    c
+# Levels: a b c
 
 \dontrun{
```

Comment:
This section will fail for package check using `--run-dontrun`, we are not using non-suggested packages anywhere in the code, including dontrun sections.
I would stick with putting `hutils`, `dplyr` chunks into NEWS.md, and keeping fifelse examples focused on fifelse rather than its alternatives (+base).

Reply:
Maybe we shall remove the `hutils` and `dplyr` call in the coalesce.Rd `dontrun` section as well? If yes, can it be included in your PR?

Reply:
@2005m yes, I will do new PR for that.